app/javascript/flavours/glitch/styles/components/status.scss

```diff
@@ -176,7 +176,8 @@
 .notif-cleaning {
   .status,
   .notification-follow,
-  .notification-follow-request {
+  .notification-follow-request,
+  .notification-admin-sign-up {
```

Comment:
I think this could actually be simplified by using `.notification` instead and would work with all types of notifications

Reply:
Went ahead and made the change